script.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const API_URL = 'https://opentdb.com/api.php?amount=10&type=multiple';
+const title = document.getElementById('title');
+const genre = document.getElementById('genre');
+const difficulty = document.getElementById('difficulty');
+const question = document.getElementById('question');
+const stBtn = document.getElementById('st-btn');
+const myAnswers = document.getElementById('my-answers');
+
+class Quiz {
+    constructor(quizData) {
+        this.quizzes = quizData.results;
+        this.correctAnswerNum = 0;
+    }
+    getQuizCategory(index) {
+        return this.quizzes[index - 1].category;
+    }
+    getQuizDifficulty(index) {
+        return this.quizzes[index - 1].difficulty;
+    }
+    getQuizQuestion(index) {
+        return this.quizzes[index - 1].question;
+    }
+    getNumQuiz() {
+        return this.quizzes.length;
+    }
+    getCorrectAnswer(index) {
+        return this.quizzes[index - 1].correct_answer;
+    }
+    getIncorrectAnswers(index) {
+        return this.quizzes[index - 1].incorrect_answers;
+    }
+    countCorrectAnswerSum(index, answer) {
+        const correctAnswer = this.quizzes[index - 1].correct_answer;
+        if (answer === correctAnswer) {
+            return this.correctAnswerNum ++;
+        }
+    }
+    getCorrectAnswerSum() {
+        return this.correctAnswerNum;
+    }
+};
+
+//開始ボタンを押した時の動き
+stBtn.addEventListener('click', () => {
+    stBtn.hidden = true;
+    fetchQuizData(1);
```

Comment:
`async` な関数、メソッドを呼び出すなら `await` をつけるようにしましょう。
```suggestion
    await fetchQuizData(1);
```